app/actions/UserActions.ts

```diff
@@ -443,6 +443,35 @@ export function sendStudentConfirmationEmail(user: string): Thunk<void> {
     },
   });
 }
+
+export function startStudentAuth(): Thunk<void> {
+  return callAPI({
+    types: User.SEND_STUDENT_CONFIRMATION_TOKEN,
+    endpoint: `/oidc/authorize/`,
+    method: 'GET',
+    meta: {
+      errorMessage: 'Studentstatus feilet',
+    },
+  });
+}
+
+export function confirmStudentAuth({
+  code,
+  state,
+}: {
+  code: string;
+  state: string;
+}): Thunk<void> {
+  return callAPI({
+    types: User.SEND_STUDENT_CONFIRMATION_TOKEN,
+    endpoint: `/oidc/validate/?code=${code}&state=${state}`,
+    method: 'GET',
```

Comment:
I'm considering changing this to a POST. There isn't really much practical difference here, more of a design choice.

Reply:
I think keeping it as a `GET` is fine